src/components/Authentication/LoginPage.js

```diff
@@ -14,6 +14,8 @@ import Typography from "@material-ui/core/Typography";
 import InputAdornment from "@material-ui/core/InputAdornment";
 import EmailOutlinedIcon from "@material-ui/icons/EmailOutlined";
 
+import { ResponsiveButton } from '../../theme/ThemedComponents/Button/ResponsiveButton';
```

Comment:
Some of your import statement use relative directories ../../theme others just start at theme/ We should keep this consistent throughout and ask Daniel which would be preferred